src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchElementViewModel.cs

```diff
@@ -21,7 +21,46 @@ public class PackageManagerSearchElementViewModel : BrowserItemViewModel, IEquat
         public ICommand VisitRepositoryCommand { get; set; }
         public ICommand DownloadLatestToCustomPathCommand { get; set; }
 
-        public new PackageManagerSearchElement Model { get; internal set; }
+        /// <summary>
+        /// The PackageManagerSearchElement Model this VM links to
+        /// </summary>
+        public PackageManagerSearchElement SearchElementModel { get; internal set; }
+        /// <summary>
+        /// VM IsDeprecated property
+        /// </summary>
+        public bool IsDeprecated { get; set; }
```

Comment:
so why do we need all these now? Were we binding to the model directly before?

Reply:
We were binding directly to the Model. We had a discussion with @QilongTang a while back to move the binding to the VM, but left it for another day. I took the opportunity to do it with this PR.